randomMessageGenerator/script.js

```diff
@@ -0,0 +1,36 @@
+const nouns = ["pasta", "movie", "music", "species", "cake", "color", "island", "salt", "umbrella", "bathtub", "meeting"];
+const adjectives = ["new", "expensive", "rich", "pink", "futuristic", "free", "important", "ugly", "spicey"];
+const verbs = ["create","invent","discover","imagine","notice","complain about","wonder about", "think about"];
+
+function getRandom(wordCollection) {
+    if (!wordCollection.length) return "";
+    else return wordCollection[Math.floor(Math.random() * wordCollection.length)];
```

Comment:
they did cover ternary expressions actually. I just try not to use it so much for future readibility reasons and frankly because I disagree on the "beautiful" 😅😅
But I get that there are certain simple cases where it makes sense to use them. Like here.